mobile/lib/shared/providers/server_info.provider.dart

```diff
@@ -15,13 +15,36 @@ class ServerInfoNotifier extends StateNotifier<ServerInfoState> {
               patch_: 0,
               minor: 0,
             ),
+            serverFeatures: ServerFeaturesDto(
+              clipEncode: true,
+              configFile: false,
+              facialRecognition: true,
+              map: true,
+              oauth: false,
+              oauthAutoLaunch: false,
+              passwordLogin: true,
+              search: true,
+              sidecar: true,
+              tagImage: true,
+            ),
+            serverConfig: ServerConfigDto(
+              loginPageMessage: "",
+              mapTileUrl: "https://tile.openstreetmap.org/{z}/{x}/{y}.png",
+              oauthButtonText: "",
+            ),
             isVersionMismatch: false,
             versionMismatchErrorMessage: "",
           ),
         );
 
   final ServerInfoService _serverInfoService;
 
+  getServerInfo() async {
+    await getServerVersion();

```

Comment:
no need for async or any await here - the calls can happen asynchronously in parallel.

Reply:
Updated!